instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/rpc/RpcClientMetricsTest.java

```diff
@@ -64,18 +64,77 @@ void collectsMetrics() {
                         TraceFlags.getSampled(),
                         TraceState.getDefault())));
 
-    Context context1 = listener.onStart(parent, requestAttributes, nanos(100));
+    Context context1 =
+        RpcMetricsHolder.init(listener.onStart(parent, requestAttributes, nanos(100)));
 
     assertThat(metricReader.collectAllMetrics()).isEmpty();
 
     Context context2 = listener.onStart(Context.root(), requestAttributes, nanos(150));
 
     assertThat(metricReader.collectAllMetrics()).isEmpty();
 
+    RpcMetricsHolder.setRequestBodySize(context1, 10);
+    RpcMetricsHolder.setResponseBodySize(context1, 20);
     listener.onEnd(context1, responseAttributes1, nanos(250));
 
     assertThat(metricReader.collectAllMetrics())
         .satisfiesExactlyInAnyOrder(
+            metric ->
+                assertThat(metric)
+                    .hasName("rpc.client.response.size")
+                    .hasUnit("By")
+                    .hasDescription("Measures the size of RPC response messages (uncompressed).")
+                    .hasHistogramSatisfying(
+                        histogram ->
+                            histogram.hasPointsSatisfying(
+                                point ->
+                                    point
+                                        .hasSum(20 /* bytes */)
+                                        .hasAttributesSatisfying(
```

Comment:
Unless there is a reason not to (for example, many unrelated attributes we don't necessarily care about), we typically use the more stricter assertion
```suggestion
                                        .hasAttributesSatisfyingExactly(
```